src/MSBuild/TerminalLogger/AnsiCodes.cs

```diff
@@ -95,13 +95,23 @@ internal static class AnsiCodes
     public const string ShowCursor = "\x1b[?25h";
 
     /// <summary>
-    /// Set progress state to a busy spinner.
+    /// Set progress state to a busy spinner. <br/>
+    /// Note: this code works only on ConEmu terminals, and conflicts with push a notification code on iTerm2.
```

Comment:
This is over-broad; several terminals support it (including most importantly to our Windows folks, Windows Terminal). I might simplify to something like "should be wrapped with a support check".

Reply:
is there any specification Windows Terminal adheres to?
I thought they adhere to ConEmu (at least to some extend).